CRM/Activity/Form/Activity.php

```diff
@@ -370,6 +370,10 @@ public function preProcess() {
       $this->assign('activityTypeName', $this->_activityTypeName);
       $this->assign('activityTypeDescription', $activityTypeDescription);
     }
+    else {
+      $this->assign('activityTypeName', FALSE);
+      $this->assign('activityTypeDescription', FALSE);
```

Comment:
My preference would be to move the assign statements outside the conditional rather than repeat them.